src/opendr/perception/object_detection_2d/nanodet/algorithm/nanodet/model/fpn/fpn.py

```diff
@@ -84,17 +87,10 @@ def forward(self, inputs):
         # build top-down path
         used_backbone_levels = len(laterals)
         for i in range(used_backbone_levels - 1, 0, -1):
-            laterals[i - 1] += F.interpolate(
+            laterals[i - 1] = laterals[i - 1] + F.interpolate(
```

Comment:
```suggestion
            laterals[i - 1] += F.interpolate(
```

Reply:
In Onnx with pytorch documentation it has a comment to Avoid in-place operations in some scenarios. I am not sure if it is ok in this particular senario so I change it.